app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] The devise gem methods aren't processing properly when I run the project locally, it completes the queries successfully. However, it cannot complete the user login to the application! Kindly double-check your implementation. 🙏

Reply:
it was working on my local machine, the issue was raised because it was not sending mail which was happening in development